__tests__/utils.ts

```diff
@@ -0,0 +1,14 @@
+
+import puppeteer from 'puppeteer';
+
+export async function refillTextInput (handle: puppeteer.ElementHandle<Element>, text: string): Promise<puppeteer.ElementHandle<Element>> {
```

Comment:
Wondering what this is about exactly; some kind of necessary hack to make puppeteer play nice?

Reply:
We need to clear input value from test to test, but Puppeteer doesn't provide direct access to DOM. Clearing input value has to be done manually, ie triple-click to select and press back button. This utility just saves typing.